### ISSUE TITLE
keepalive.. per backend? 

### ISSUE THREAD

**Issue by bcookatpcsd** [NginxProxyManager/nginx-proxy-manager#4162]
tl;dr: (how do I correctly add keepalive_timeout to my backend..)

Everything is working, 6 hosts working as configured. 

Looking to make the 'unifi' controller app more responsive.. 

I see in nginx.conf 

http{

..

keepalive_timeout 90s;

..

but in 

/data/nginx/proxy_host/

[1-6].conf I do not see keepalive_timeout set in each backend.. 

In the nginx docs keepalive has no default but keepalive_timeout has 60s default.. but it's in the context of upstream

so I create /data/custom/server_proxy.conf and added:

keepalive 2;
keepalive_timeout 90s;

and restarted the app..

am I missing something? 

Or is something missing.. 

![image](https://github.com/user-attachments/assets/423405fd-7ab6-4f04-8471-f2550a4032d4)

![image](https://github.com/user-attachments/assets/2101c088-db1d-4afc-a39e-0368909c076b)

Where is server_proxy.conf defined to be used? 

Thank you in advance..

**Comment by bcookatpcsd** [comment]
missed including this.. 

```
tcpdump -v -i eno1.69 -n -q dst host 10.20.8.29 -A

.bP...?)..GET /speedtest.js HTTP/1.1
Host: files.tech.domain.internal
X-Forwarded-Scheme: http
X-Forwarded-Proto: http
X-Forwarded-For: 10.120.148.95
X-Real-IP: 10.120.148.95
Connection: close
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:132.0) Gecko/20100101 Firefox/132.0
Accept: */*
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate
Sec-GPC: 1
Referer: http://files.tech.domain.internal/
Priority: u=2

```
